jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java

```diff
@@ -42,6 +42,7 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 
+@SuppressWarnings("all")
```

Comment:
it's bad to supress all warnings, why do we need to supress all?

Reply:
@SuppressWarnings("PMD.NPathComplexity")
@SuppressWarnings({"checkstyle:npathcomplexity", "checkstyle:cyclomaticcomplexity"})
possibly someone of that can help us

Reply:
When I suppress all warnings complexity decreases only 10. The same was with attempt of suppressing for all class. I looked through code and don't see chance to suppress more functions used in getDriver() as it's read-only. So here is question for suppressing another way and deleting all my changes. It was discussed on stand up. 
To see result of suppress for each commit you could go https://app.codacy.com/.

Reply:
> 
> 
> @SuppressWarnings("PMD.NPathComplexity")
> @SuppressWarnings({"checkstyle:npathcomplexity", "checkstyle:cyclomaticcomplexity"})
> possibly someone of that can help us

Guess `@SuppressWarnings("PMD.NPathComplexity")` above the method should work right